docs/getting-started/aspnet5.rst

```diff
@@ -1,5 +1,3 @@
-.. include:: /stub-topic.txt
-
```

Comment:
Yeah it was left over from when this article wasn't written (this include just makes the icon available for use... it doesn't actually render any content).